java/src/main/java/com/mlt/converter/triangulation/TriangulationUtils.java

```diff
@@ -0,0 +1,76 @@
+package com.mlt.converter.triangulation;
+
+import earcut4j.Earcut;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.ArrayUtils;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.MultiPolygon;
+import org.locationtech.jts.geom.Polygon;
+
+public class TriangulationUtils {
+  private TriangulationUtils() {}
+
+  public static TriangulatedPolygon triangulatePolygon(Polygon polygon) {
```

Comment:
A `Polygon` can have holes, and a `MultiPolygon` just contains many `Polygon`s (which may each have holes). Can we change this to handle those cases and use the `poly.getExteriorRing()` `poly.getNumInteriorRing()` `poly.getInteriorRingN(int n)` helpers? Let's add some tests for polygons with holes as well.